### DISCUSSION TITLE
Multibox detection negative anchor width

### DISCUSSION THREAD

**Discussion by matanhs** [apache/incubator-mxnet#12452] [Q&A]
Hi, 
Please correct me if I am wrong, It seems the marked line has a small bug that makes the anchor height negative (as translated from the anchor corners before the exponent assuming the anchors are in order the of [xmax,ymin,xmin,ymax])
https://github.com/apache/incubator-mxnet/blob/d00cb42458802e4471c9517720924f2d595e02c0/src/operator/contrib/multibox_detection.cc#L56

Thanks!

**Comment by vrakesh** [comment]
Thank you for the question @matanhs , we will get back to you on this.

@mxnet-label-bot [Question, Possible Bug]
